front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -111,8 +124,10 @@ const TimesStops = ({ pathProperties, pathSteps = [] }: TimesStopsProps) => {
           checkboxColumn as Partial<Column<boolean | undefined>>
         ),
         title: t('receptionOnClosedSignal'),
+
+        // We should not be edit the reception on close signal of the origin
         grow: 0.6,
-        disabled: ({ rowData }) => !rowData.stopFor,
+        disabled: ({ rowData, rowIndex }) => !rowData.stopFor || rowIndex === 0,
```

Comment:
As we can't edit the stopping time of the origin, we won't be able to edit the "on close "signal" anyway so I don't think we need this condition (if you agree, you can also remove the comment)

Reply:
Done